### ISSUE TITLE
ERROR: there is already one or more instance(s) of the program running

### ISSUE THREAD

**Issue by bwitt** [brightroll/rq#76]
If RQ dies without getting to clean itself up (e.g. the host locks up and is reset), subsequent attempts to start RQ are met with

```
# /etc/init.d/rq  start
ERROR: there is already one or more instance(s) of the program running
```

Workaround is to remove the pid file and retry:

```
# rm /rq/current/config/queuemgr.pid  
# /etc/init.d/rq  start
#
```

this is with 

```
RQ_VER = '1.17.2'
```

**Comment by bwitt** [comment]
can't reproduce on 
```
RQ_VER = '1.18.1'
```
so I'm closing

**Comment by sodabrew** [comment]
This should be fixed in RQ v1.18 with the update to the daemons gem.
